malduck/crypto/des3.py

```diff
@@ -1,20 +1,21 @@
 # Copyright (C) 2018 Jurriaan Bremer.
 # This file is part of Roach - https://github.com/jbremer/roach.
 # See the file 'docs/LICENSE.txt' for copying permission.
-from typing import Any
+from typing import cast
 
 from Cryptodome.Cipher import DES, DES3 as DES3Cipher
+from Cryptodome.Cipher._mode_cbc import CbcMode
```

Comment:
Looking at this first time I was not sure whether we want to provide typings for that `get_cipher`

`CbcMode` looks to be a private Cryptodome module and not a part of stable interface so it's possible that it will be removed in further version.

Reply:
While that's true, `_get_cipher` is a private method that is not exposed outside of the interface :thinking: